### ISSUE TITLE
Widevine DRM content not playing on emulator, how do I specify tizen permissions?

### ISSUE THREAD

**Issue by husseinmkwizu** [pavjacko/renative#562]
I am developing  a player for our Smart TV app. I have done the integrations with widevine DRM using shaka player, they are working fine in Firefox and Chrome using. "rnv run -p tine --hosted", the issue is when I run it on emulator I am not able to play the video streams. Any ideas why?

My guess is I need to add permissions for DRM, as in this Samsung DRM example

https://github.com/SamsungDForum/PlayerAVPlayDRM/blob/master/PlayerAVPlayDRM/config.xml

The config.xml generated by renative, doesn't include DRM permissions.

How do I add permissions to tizen in renative?

Thanks.

**Comment by ali-sao** [comment]
Hello @husseinmkwizu , I don't think it is possible to play DRM on emulator.

**Comment by husseinmkwizu** [comment]
Thanks for the reply. can you describe how I can add permissions then? Because that will be required even on a real device

**Comment by ali-sao** [comment]
@husseinmkwizu the Samsung DRM example is all about it buddy.

**Comment by husseinmkwizu** [comment]
What I mean is the keys and values I can use in renative.json for tizen. something like this

https://renative.org/docs/api-config#permissions

I have tried to edit the config.xml generated by renative at platformBuilds/PROJECT_NAME/config.xml, but it seems like renative is generating it's own and overwriting the config I wrote.

So I was wondering which keys I should use on the renative.json, so that the generated config.xml has the permissions required.

Thanks again for the reply.
